Lib/fontTools/subset/cff.py

```diff
@@ -478,36 +398,11 @@ def _delete_empty_subrs(private_dict):
 			del private_dict.rawDict['Subrs']
 		del private_dict.Subrs
 
+
+# TODO: This is now unused by this module, should we deprecate it?
```

Comment:
yes, we should deprecate it. Please try use this decorator from 
https://github.com/fonttools/fonttools/blob/9abfc00b71fc2d1aebe5f9f52ce737dac0fe7c05/Lib/fontTools/misc/loggingTools.py#L521

Reply:
Done.